codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/ValueEntitySourceGeneratorSuite.scala

```diff
@@ -44,4 +44,29 @@ class ValueEntitySourceGeneratorSuite extends munit.FunSuite {
          |}
          |""".stripMargin)
   }
+  test("it can generate an abstract value entity implementation") {
+    val str = abstractEntity(TestData.valueEntity(domainParent), TestData.simpleEntityService(apiParent))
+    assertNoDiff(
+      str,
+      s"""package com.example.service.domain
+          |
+          |import com.akkaserverless.scalasdk.valueentity.ValueEntity
+          |import com.akkaserverless.scalasdk.valueentity.ValueEntity.Effect
+          |import com.example.service.GetValue
+          |import com.example.service.MyState
+          |import com.example.service.SetValue
+          |import com.external.Empty
```

Comment:
Ive copied this from other tests, why is this not `google.protobuf.Empty`? @patriknw @raboof @octonato

Reply:
I guess we wanted an example of referring to an external proto file?

Reply:
yeah, that's the reason